src/Mobs/Monster.cpp

```diff
@@ -1073,9 +1073,9 @@ cMonster * cMonster::NewMonsterFromType(eMonsterType a_MobType)
 		case mtHorse:
 		{
 			// Horses take a type (species), a colour, and a style (dots, stripes, etc.)
-			int HorseType = Random.NextInt(7);
-			int HorseColor = Random.NextInt(6);
-			int HorseStyle = Random.NextInt(6);
+			int HorseType = Random.NextInt(8);
```

Comment:
Yeah and the types 5, 6, 7 are used to increase spawning rate of the normal horses.
In the next lines is an if, which maps the placeholder values to 0(normal horse).